pandas/core/indexes/datetimes.py

```diff
@@ -2237,6 +2238,22 @@ def date_range(start=None, end=None, periods=None, freq='D', tz=None,
     Returns
     -------
     rng : DatetimeIndex
+
+    See Also
+    ________
+    pandas.period_range : Return a fixed frequency PeriodIndex.
+    pandas.interval_range : Return a fixed frequency IntervalIndex.
+
+    Examples
+    --------
+    >>> import pandas
+    >>> pandas.date_range('20181003', periods=2)
```

Comment:
Can you do these in iso format `2018-10-03`?

Reply:
will do it

Reply:
Done!